src/heranca_exemplos/marcas_de_veiculos/Base.java

```diff
@@ -12,94 +10,94 @@ public class Base {
 	protected String marca;
 	protected String nome;
 	protected String tipoDeVeiculo;
-	protected int quantidadeDePortas;
-	protected int quantidadeDeRodas;
-	public static int veiculoEmOrdem = 0;
-	protected List<String> veiculosEsquematizados = new ArrayList<String>();
-	protected List<String> veiculosConstruidos = new ArrayList<String>();
-	protected Map<String, String> concessionarias = new HashMap<String, String>();
-	private int quantidadeDeMateriaisDisponiveis = 30, materiais = 8;
-
-	public void esquematizaModelo(String marca, String nome, String tipoDeVeiculo, int quantidadeDePortas,
-			int quantidadeDeRodas) {
-		this.tipoDeVeiculo = tipoDeVeiculo;
-		this.quantidadeDePortas = quantidadeDePortas;
-		this.quantidadeDeRodas = quantidadeDeRodas;
+	protected int kmPorLitro;
+	protected int velocidade;
+	protected int valor;
+	public static int veiculoEmOrdem = 1;
```

Comment:
Sobre o atributo abaixo

public static int veiculoEmOrdem = 1;

Se for usado apenas na classe Base pode ser privado
Se for usado nas heranças da classe base pode ser protected
Só deve ser public se realmente qualquer outra classe em qualquer outro pacote do projeto vai invocar